assets/js/custom.js

```diff
@@ -204,7 +204,9 @@ $(function() {
 
 $("#hidecookie").click(function () {
   $(".cookie-block").slideUp("slow");
-  createCookie('greenpeace', 'policy-accepted');
+  var expiryDate = new Date();
+  expiryDate.setMonth(expiryDate.getMonth() + 12);
```

Comment:
This creates a Date() js object.
What you need to pass in your function is a number of days. 
This does not work. Fix it.